crates/top/re_sdk/src/recording_stream.rs

```diff
@@ -1030,6 +1030,41 @@ impl RecordingStream {
         Ok(())
     }
 
+    /// Lower-level logging API to provide data spanning multiple timepoints.
+    ///
+    /// Unlike the regular `log` API, which is row-oriented, this API lets you submit the data
+    /// in a columnar form. The lengths of all of the [`TimeColumn`] and the component columns
+    /// must match. All data that occurs at the same index across the different time and components
+    /// arrays will act as a single logical row.
+    ///
+    /// Note that this API ignores any stateful time set on the log stream via the
+    /// [`Self::set_timepoint`]/[`Self::set_time_nanos`]/etc. APIs.
+    /// Furthermore, this will _not_ inject the default timelines `log_tick` and `log_time` timeline columns.
+    pub fn send_columns_v2(
```

Comment:
I'll rename this to `send_columns` and remove all the legacy stuff in a follow-up, I don't want the diffs to be unnecessarily hard to follow.